bgpd/bgp_vty.c

```diff
@@ -100,6 +100,12 @@ FRR_CFG_DEFAULT_ULONG(BGP_KEEPALIVE,
 	{ .val_ulong = 3, .match_profile = "datacenter", },
 	{ .val_ulong = 60 },
 )
+FRR_CFG_DEFAULT_BOOL(BGP_EBGP_REQUIRES_POLICY,
```

Comment:
2 nitpicks :)

- can you match the style of the other FRR_CFG_DEFAULT_*?
- the default should only change for version >= 7.4, you get that by doing:

```
{ .val_bool = false, .match_profile = "datacenter", },
{ .val_bool = false, .match_version = "<= 7.4", },
{ .val_bool = true, },
```

This makes it so that a config created / saved in 7.3 will load with the correct old default (and it will be written out as a visible statement in "show run")